curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxs.md

```diff
@@ -0,0 +1,122 @@
+---
+id: 5c6c06847491271903d37cfd
+title: Use the value attribute with Radio Buttons and Checkboxes
+challengeType: 0
+---
+
+## Description
+<section id='description'>
+When our form gets submitted the form data sent to the server includes entries identifying the choices and input values. Inputs of type <code>radio button</code> and <code>checkbox</code> reports the value given to the value attribute.
```

Comment:
```suggestion
When our form gets submitted. the form data sent to the server includes entries identifying the choices and input values. Inputs of type <code>radio button</code> and <code>checkbox</code> reports the value given to the value attribute.
```

Reply:
I'm confused, did you mean you want a comma here?

Reply:
Yes, sorry, it should be a comma

Reply:
I've updated my comment accordingly